src/organisms/cards/PlaybackCardWrapper/Readme.md

```diff
@@ -49,31 +45,27 @@ PlaybackCardWrapper Example:
         onClick={() => alert('edit clicked')}
         leftPanel={
           <div>
-            <TextLockup
-              variant='G'
-              header='Name'
-              subheader='Victoria Bangladesh'
+            <TextD
+              line1='Name'
+              line2='Victoria Bangladesh'
             />
 
-            <TextLockup
-              variant='G'
-              header='Email'
-              subheader='victoriabangladesh19283@gmail.com'
+            <TextD
+              line1='Email'
+              line2='victoriabangladesh19283@gmail.com'
             />
           </div>
         }
 
         rightPanel={
           <div>
-            <TextLockup
-              variant='G'
-              header='Date of Birth'
-              subheader='9/19/1987'
+            <TextD
```

Comment:
@patrickkim Yeah -- do you feel strongly about this explicitly named TextLockup components convention?